eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/service/config/synchronous/SynchronousConnectorConfigTask.java

```diff
@@ -0,0 +1,91 @@
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.dashboard.console.service.config.synchronous;
+
+import org.apache.eventmesh.dashboard.console.entity.config.ConfigEntity;
+import org.apache.eventmesh.dashboard.console.entity.connector.ConnectorEntity;
+import org.apache.eventmesh.dashboard.console.service.config.ConfigService;
+import org.apache.eventmesh.dashboard.console.service.config.instanceoperation.ConnectorConfigController;
+import org.apache.eventmesh.dashboard.console.service.connector.ConnectorDataService;
+
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SynchronousConnectorConfigTask {
+
+    @Autowired
+    private ConnectorDataService connectorDataService;
+
+    @Autowired
+    private ConnectorConfigController connectorConfigController;
+    @Autowired
+    private ConfigService configService;
+
+    public void synchronousConnectorConfig(Long clusterId) {
+        List<ConnectorEntity> connectorEntities = connectorDataService.selectConnectorByCluster(clusterId);
+        for (ConnectorEntity connectorEntity : connectorEntities) {
+
+            ConcurrentHashMap<String, String> connectorConfigMapFromInstance = this.configListToMap(
+                connectorConfigController.getConnectorConfigFromInstance(clusterId, connectorEntity.getId()));
+
+            ConfigEntity configEntity = this.getConfigEntityBelongInstance(clusterId, connectorEntity.getId());
+
+            ConcurrentHashMap<String, String> connectorConfigMapFromDb = this.configListToMap(configService.selectByInstanceId(configEntity));
+
+            ConcurrentHashMap<String, String> updateConfigMap = new ConcurrentHashMap<>();
+
+            connectorConfigMapFromInstance.entrySet().forEach(n -> {
+                if (connectorConfigMapFromDb.remove(n.getKey(), n.getValue())) {
+                    connectorConfigMapFromInstance.remove(n.getKey());
+                }
+                if (connectorConfigMapFromDb.get(n.getKey()) != null) {
+                    updateConfigMap.put(n.getKey(), connectorConfigMapFromDb.get(n.getKey()));
+                    connectorConfigMapFromInstance.remove(n.getKey());
+                    connectorConfigMapFromDb.remove(n.getKey());
+                }
+            });
+            //add  connectorConfigMapFromDb
+
+            //update  updateConfigMap
+
+            //delete connectorConfigMapFromInstance
+        }
+    }
+
+    public ConcurrentHashMap<String, String> configListToMap(List<ConfigEntity> configEntityList) {
+        ConcurrentHashMap<String, String> connectorConfigMap = new ConcurrentHashMap<>();
+        configEntityList.forEach(n -> {
+                connectorConfigMap.put(n.getConfigName(), n.getConfigValue());
+            }
+        );
+        return connectorConfigMap;
+    }
+
```

Comment:
Utility methods should be decorated with `private` namespace or moved to util class.